### ISSUE TITLE
Vulnerabilities found on v1.4.0

### ISSUE THREAD

**Issue by raafatseif** [FairwindsOps/rbac-manager#343]
Hi team!

**Is your feature request related to a problem? Please describe.**
Would like to see if we can update dependencies and tools on the base image (specifically zlib) to get rid of some vulnerabilities. See trivy report below for `rbac-manager:v1.4.0` image:

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
    -v $HOME/Library/Caches:/root/.cache/ docker.nexus.heb.tools/aquasec/trivy:0.32.1 image quay.io/reactiveops/rbac-manager:v1.4.0
2022-10-14T15:28:04.486Z        INFO    Need to update DB
2022-10-14T15:28:04.486Z        INFO    DB Repository: ghcr.io/aquasecurity/trivy-db
2022-10-14T15:28:04.486Z        INFO    Downloading DB...
2022-10-14T15:28:30.177Z        INFO    Secret scanning is enabled
2022-10-14T15:28:30.177Z        INFO    If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2022-10-14T15:28:30.177Z        INFO    Please see also https://aquasecurity.github.io/trivy/v0.32/docs/secret/scanning/#recommendation for faster secret detection
2022-10-14T15:28:33.046Z        INFO    Detected OS: alpine
2022-10-14T15:28:33.046Z        INFO    Detecting Alpine vulnerabilities...
2022-10-14T15:28:33.048Z        INFO    Number of language-specific files: 1
2022-10-14T15:28:33.048Z        INFO    Detecting gobinary vulnerabilities...

quay.io/reactiveops/rbac-manager:v1.4.0 (alpine 3.16.1)
=======================================================
Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 1)

┌─────────┬────────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────────┐
│ Library │ Vulnerability  │ Severity │ Installed Version │ Fixed Version │                            Title                            │
├─────────┼────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────────┤
│ zlib    │ CVE-2022-37434 │ CRITICAL │ 1.2.12-r1         │ 1.2.12-r2     │ zlib: heap-based buffer over-read and overflow in inflate() │
│         │                │          │                   │               │ in inflate.c via a...                                       │
│         │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2022-37434                  │
└─────────┴────────────────┴──────────┴───────────────────┴───────────────┴─────────────────────────────────────────────────────────────┘

rbac-manager (gobinary)
=======================
Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 1, CRITICAL: 0)

┌───────────────────┬────────────────┬──────────┬────────────────────────────────────┬───────────────────────────────────┬─────────────────────────────────────────────────────────────┐
│      Library      │ Vulnerability  │ Severity │         Installed Version          │           Fixed Version           │                            Title                            │
├───────────────────┼────────────────┼──────────┼────────────────────────────────────┼───────────────────────────────────┼─────────────────────────────────────────────────────────────┤
│ golang.org/x/net  │ CVE-2022-27664 │ HIGH     │ v0.0.0-20220607020251-c690dde0001d │ 0.0.0-20220906165146-f3363e06e74c │ golang: net/http: handle server errors after sending GOAWAY │
│                   │                │          │                                    │                                   │ https://avd.aquasec.com/nvd/cve-2022-27664                  │
├───────────────────┼────────────────┼──────────┼────────────────────────────────────┼───────────────────────────────────┼─────────────────────────────────────────────────────────────┤
│ golang.org/x/text │ CVE-2022-32149 │ MEDIUM   │ v0.3.7                             │ 0.3.8                             │ golang: golang.org/x/text/language: ParseAcceptLanguage     │
│                   │                │          │                                    │                                   │ takes a long time to parse complex tags                     │
│                   │                │          │                                    │                                   │ https://avd.aquasec.com/nvd/cve-2022-32149                  │
└───────────────────┴────────────────┴──────────┴────────────────────────────────────┴───────────────────────────────────┴─────────────────────────────────────────────────────────────┘
```

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by raafatseif** [comment]
Hi @sudermanjr ! How does this issue look? Just wanted to know if I need to add any more details

**Comment by sudermanjr** [comment]
Hey there, thanks for reporting. I just pushed a tag to release 1.4.1

**Comment by sudermanjr** [comment]
The 1.4.1 release fixes the zlip vulnerability, and I opened #354 to handle the others

**Comment by sudermanjr** [comment]
Just pushed v1.4.2

```
trivy i quay.io/reactiveops/rbac-manager:v1.4.2
2022-11-17T13:14:07.108-0700    INFO    Detected OS: alpine
2022-11-17T13:14:07.108-0700    INFO    This OS version is not on the EOL list: alpine 3.16
2022-11-17T13:14:07.108-0700    INFO    Detecting Alpine vulnerabilities...
2022-11-17T13:14:07.112-0700    INFO    Number of language-specific files: 1
2022-11-17T13:14:07.112-0700    INFO    Detecting gobinary vulnerabilities...

quay.io/reactiveops/rbac-manager:v1.4.2 (alpine 3.16.3)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


rbac-manager (gobinary)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
